src/scry/implementations.cr

```diff
@@ -39,6 +39,10 @@ module Scry
         end
         ResponseMessage.new(@text_document.id, locations)
       end
+    rescue ex
+      nil
```

Comment:
Should we at least log that an exception happened? What kind of exceptions are we expecting here?